src/main/webapp/feedback/load-icon-feedback.js

```diff
@@ -91,6 +92,89 @@ export default class LoadIconFeedback {
           TimestampUtil.millisecondsToSeconds(intervalLowerBound),
     };
   }
+
+  /** Charts IconFeedback data into a graph. */
```

Comment:
I saw in the screenshot it was for milliseconds. I think it should be the same format as the video hours:minutes:seconds because if the prof uploads a 2hour video it will be annoying to have to convert milliseconds to video time to see what the feedback is for.

Reply:
Yeah that would be much better, added!

Reply:
Can you update the screenshot?